PlayerControls/sources/SideBarView.swift

```diff
@@ -135,7 +138,7 @@ public final class SideBarView: UIView {
                 button.setImage(prop.icons.highlighted, for: .highlighted)
                 button.accessibilityLabel = prop.accessibility.label
                 button.accessibilityHint = prop.accessibility.hint
-                button.accessibilityTraits = prop.accessibility.traits
+                button.accessibilityTraits = UIAccessibilityTraits(rawValue: UIAccessibilityTraits.button.rawValue)
```

Comment:
It's the same as `UIAccessibilityTraits.button`

Reply:
Yeah, works on 4.0